### ISSUE TITLE
small bugs

### ISSUE THREAD

**Issue by asherikov** [PickNikRobotics/data_tamer#41]
Inconsistent cmake namespace: when building without ROS, the namespace is `data_tamer::` instead of `data_tamer_cpp::`, in general all `data_timer` strings in https://github.com/PickNikRobotics/data_tamer/blob/main/data_tamer_cpp/CMakeLists.txt#L141 should be replaced with ${PROJECT_NAME}.

Package manifest lacks mcap build dependency -- liblz4-dev.

**Comment by jlack1987** [comment]
I see the same thing, this is I guess technically an issue with the 3rd party mcap library though

**Comment by asherikov** [comment]
Yes, but the flag is set here https://github.com/PickNikRobotics/data_tamer/blob/main/data_tamer_cpp/CMakeLists.txt#L83

**Comment by facontidavide** [comment]
The missing dependencies were declared in #76 (`lz4` and `libzstd-dev` in package.xml). The CMake namespace stays `data_tamer::`: renaming it to `data_tamer_cpp::` would break every downstream `target_link_libraries(... data_tamer::data_tamer)`, PlotJuggler included. If a project-name namespace is wanted it should be added as an alias target next to the existing one, not as a rename.
